src/shared/components/common/password-input.tsx

```diff
@@ -97,8 +97,8 @@ class PasswordInput extends Component<PasswordInputProps, PasswordInputState> {
                 onInput={onInput}
                 value={value}
                 required
-                maxLength={60}
-                minLength={10}
+                pattern=".{10,60}"
+                title={I18NextService.i18n.t("invalid_password")}
```

Comment:
See [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern#usability_and_accessibility_considerations) for why I'm using the `title` prop for this.